src/SDKs/_metadata/peering_resource-manager.txt

```diff
@@ -0,0 +1,10 @@
+2020-01-23 19:30:24 UTC
```

Comment:
This file should be in `/eng/mgmt/mgmtmetadata/`
Please make sure you generate your code by running `generate.ps1` (not autorest directly). And if the problem still goes on, try [updating build tools](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools)